### ISSUE TITLE
ValueError: Unknown format code 'd' for object of type 'float'

### ISSUE THREAD

**Issue by nnaoumov** [sebdah/dynamic-dynamodb#265]
I just updated dynamic_dynamodb with pip and got version 2.0 and now am seeing this error. I noticed that logged capacities are now decimal percentages while in the old version they were ints - e.g. dynamic-dynamodb - INFO - DeviceMapping - Consumed read units: 75.17%

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/dynamic_dynamodb/__init__.py", line 98, in main
    execute()
  File "/usr/local/lib/python2.7/dist-packages/dynamic_dynamodb/__init__.py", line 131, in execute
    table_num_consec_write_checks)
  File "/usr/local/lib/python2.7/dist-packages/dynamic_dynamodb/core/table.py", line 36, in ensure_provisioning
    __ensure_provisioning_alarm(table_name, key_name)
  File "/usr/local/lib/python2.7/dist-packages/dynamic_dynamodb/core/table.py", line 871, in __ensure_provisioning_alarm
    reads_upper_alarm_threshold))
ValueError: Unknown format code 'd' for object of type 'float'
```

This is the latest package in pip:
```
pip show dynamic-dynamodb
---
Name: dynamic-dynamodb
Version: 2.0.0
Location: /usr/local/lib/python2.7/dist-packages
Requires: boto, requests, logutils, retrying
```

**Comment by PaulLiang1** [comment]
HI sebdah,

Is there a work around for current version in pip for this issue?

**Comment by sebdah** [comment]
This has now been released in version 2.0.1
